### PR TITLE
Gherkin for console dark theme

### PR DESCRIPTION
**Pull request by sanketpathak**
Epic: https://issues.redhat.com/browse/ODC-5990
Story: https://issues.redhat.com/browse/ODC-6499

Acceptance criteria:

- Console users should be able to set their theme preference in User Preferences. They should be able to select between matching OS defaults, light or dark. 

**Checks required for approving Epic gherkin scripts PR:**

- [x] Add @epic-number to the scenarios or feature file [e.g: @odc-xxx]
- [x] Add @to-do for automation possible scenarios
- [x] Add @regression or @smoke based on the importance of the scenario
- [x] Update the test scenarios count in [Automation status confluence Report](https://docs.jboss.org/display/ODC/Automation+Status+Report)
- [x] Check for the linter issues by executing `yarn run gherkin-lint` on frontend folder [Skip epic number tags related linter issues]

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/user-preferences/user-preferences-dev-perspective.feature [hunk 1]
```diff
@@ -112,3 +112,10 @@ Feature: Manage user preferences
               And user enters name as "node-route" in General section
               And user clicks "Show advanced Routing options" link in Advanced Options section
              Then user is able to see "Secure Route" checkbox is deselected
+
+
+        @regression @manual @odc-5990
+        Scenario: Setting theme preference for Console: UP-01-TC11
+             When user clicks on user dropdown on masthead and selects "User Preference"
```
Comment (vikram-raj) [inline frontend/packages/dev-console/integration-tests/features/user-preferences/user-preferences-dev-perspective.feature:119]:
```suggestion
             When user clicks on user dropdown on masthead and selects "User Preferences"
```

**Review by vikram-raj — COMMENTED** [review]
/lgtm

**Comment by sanketpathak** [comment]
Pr only affects E2E tests
/label qe-approved
/label docs-approved
/label px-approved

**Pull request merged**